src/helpers.ts

```diff
@@ -257,3 +257,50 @@ export const CACHE_PATH = join(cwd, '.synthetics', process.pid.toString());
 export function getDurationInUs(duration: number) {
   return Math.trunc(duration * 1e6);
 }
+
+export function megabytesToBytes(megabytes: number) {
+  return megabytes * 1024 * 1024;
+}
+
+export const networkConditionDefaults = {
+  download: 5, // megabytes/second
+  upload: 3, // megabytes/second
+  latency: 20, // milliseconds
+}
+
+export function parseNetworkConditions(args: string): NetworkConditions {
```

Comment:
The code works correctly, But we can make some improvements to make it easier to understand

1. We could reuse the `DEFAULT_NETWORK_CONDITIONS` as the default value for networkConditions
```
 const networkConditions = { ...DEFAULT_NETWORK_CONDITIONS };
```
2. if we find the  token defined, then use the provided value or keep using the default. 

Advantage of this is that, we dont need to populate empty object and change object shape as we go along, Instead keep the structure same and then `return networkConditions;` only if user specified any options instead of recreating another object like we do now https://github.com/elastic/synthetics/pull/383/files#diff-ca5696b367d474e785317cb7a0d9853fef5729387ab8d0fab4c46268c03aae99R298-R304